File: backend/src/main/java/softeer/wantcar/cartalog/entity/trim/Trim.java
package softeer.wantcar.cartalog.entity.trim;

import lombok.Builder;
import softeer.wantcar.cartalog.entity.model.ModelExteriorColor;
import softeer.wantcar.cartalog.entity.model.ModelInteriorColor;

import java.util.List;

@Builder
public class Trim {
    private Long id;
    private String name;
    private String description;
    private String imageUrl;
    private int minPrice;
    private int maxPrice;
    private List<ModelExteriorColor> exteriorColors;
    private List<ModelInteriorColor> interiorColors;
}

File: Android/app/src/main/java/com/softeer/cartalog/data/model/trim/Trim.kt
package com.softeer.cartalog.data.model.trim

data class Trim(
    val id: Int,
    val name: String,
    val description: String,
    val minPrice: Int,
    val maxPrice: Int,
    val exteriorImageUrl: String,
    val interiorImageUrl: String,
    val wheelImageUrl: String,
    val hmgData: ArrayList<TrimHmgData?>?,
    val defaultInfo: DefaultInfo?
) {

}
